api/src/main/java/org/openmrs/module/fhir/api/util/FHIRConditionUtil.java

```diff
@@ -65,11 +81,236 @@
 			conceptDt.setCoding(dts);
 			fhirCondition.setCode(conceptDt);
 		}
+		fhirCondition.setClinicalStatus(
+				MapOpenmrsStatusToFHIRClinicalStatus(condition));
 
 		return fhirCondition;
 	}
 
-	public static org.openmrs.Condition generateOpenMRSCondition() {
-		return null;
+	/**
+	 * generates OpenmrsCondition from FHIR Conditions
+	 *
+	 * @param condition FHIR condition
+	 * @return Openmrs Condition
+	 * @since 1.20.0
+	 */
+	public static org.openmrs.Condition generateOpenMRSCondition(org.hl7.fhir.dstu3.model.Condition condition) {
+		org.openmrs.Condition openMrsCondition;
+		List<String> errors = new ArrayList<>();
```

Comment:
What are you eventually using the errors list for?

Reply:
`openMrsCondition.setConcept(FHIRUtils.getConceptFromCode(condition.getCode(), errors));`

`FHIRUtils.checkGeneratorErrorList(errors);`

Reply:
I do not seem to see `FHIRUtils.checkGeneratorErrorList(errors)` in this method. On which line number is it?

Reply:
@corneliouzbett did you see the above?

Reply:
Check Line 129